tutorials/euclid_access/Euclid_ERO.md

```diff
@@ -0,0 +1,687 @@
+
+# Photometry tools
+import sep
+from photutils.detection import DAOStarFinder
+from photutils.psf import PSFPhotometry, IterativePSFPhotometry, CircularGaussianSigmaPRF, make_psf_model_image
+from photutils.background import LocalBackground, MMMBackground
+
+# Firefly
+from firefly_client import FireflyClient
+
+# Matplotlib
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+```
+
+Next, we define some parameters for `Matplotlib` plotting.
+
+```{code-cell} ipython3
+## Plotting stuff
+mpl.rcParams['font.size'] = 14
+mpl.rcParams['axes.labelpad'] = 7
+mpl.rcParams['xtick.major.pad'] = 7
+mpl.rcParams['ytick.major.pad'] = 7
+mpl.rcParams['xtick.minor.visible'] = True
+mpl.rcParams['ytick.minor.visible'] = True
+mpl.rcParams['xtick.minor.top'] = True
+mpl.rcParams['xtick.minor.bottom'] = True
+mpl.rcParams['ytick.minor.left'] = True
+mpl.rcParams['ytick.minor.right'] = True
+mpl.rcParams['xtick.major.size'] = 5
+mpl.rcParams['ytick.major.size'] = 5
+mpl.rcParams['xtick.minor.size'] = 3
+mpl.rcParams['ytick.minor.size'] = 3
+mpl.rcParams['xtick.direction'] = 'in'
+mpl.rcParams['ytick.direction'] = 'in'
+#mpl.rc('text', usetex=True)
+mpl.rc('font', family='serif')
+mpl.rcParams['xtick.top'] = True
+mpl.rcParams['ytick.right'] = True
+mpl.rcParams['hatch.linewidth'] = 1
+
+def_cols = plt.rcParams['axes.prop_cycle'].by_key()['color']
+```
+
+## Setting up the Environment
+
+Next, we set up the environment. This includes
+* setting up an output data directory (will be created if it does not exist)
+* define the search radius around the target of interest to pull data from the Gaia catalog
+* define the cutout size that will be used to download a certain part of the Euclid ERO images
+
+Finally, we also define the target of interest here. We can choose between two globular clusters, **NGC 6254** and **NGC6397**, both covered by the Euclid ERO data.
+
+Note that `astropy` units can be attached to the `search_radius` and `cutout_size`.
+
+```{code-cell} ipython3
+# create output directory
+if os.path.exists("./data/"):
+    print("Output directory already created.")
+else:
+    print("Creating data directory.")
+    os.mkdir("./data/")
+
+search_radius = 1.5 * u.arcmin # search radius
+cutout_size = 1.5 * u.arcmin # cutout size
+
+coord = SkyCoord.from_name('NGC 6397')
+```
+
+## Search Euclid ERO Images
+
+Now, we search for the Euclid ERO images using the `astroquery` package.
+Note that the Euclid ERO images are no in the cloud currently, but we access them directly from IRSA using IRSA's *Simple Image Access* (SIA) methods.
+
+```{note}
+The following only works for combined images (either extended or point source stacks). This would not work if there are multiple, let's say, H-band images of Euclid at a given position. Therefore, no time domain studies here (which is anyway not one of the main goals of Euclid).
+```
+
+The IRSA SIA products can be listed via
+```
+Irsa.list_collections(servicetype='SIA')
+```
```

Comment:
Maybe this should be inline, in the rendered version it confused me because I was expecting to see an output cell but I didn't see one